src/99-appendices/11-qmri.md

```diff
@@ -0,0 +1,547 @@
+| MTS                  | `FlipAngle`, `MTState`, `RepetitionTimeExcitation`                                                                           |                            |
+| MPM                  | `FlipAngle`, `MTState`, `RepetitionTimeExcitation`                                                                           | `EchoTime`                 |
+
+<sup>*</sup> Please see MP2RAGE-specific notes for the calculation of `NumberShots` and regarding the
+organization of `UNIT1` image.
+
+Explanation of the table:
+
+* The metadata fields listed in the REQUIRED column are needed to perform a minimum viable
+qMRI processing for the corresponding `file collection`.
+* Note that some of the metadata fields may be constant across different files in a file
+collection, yet still required as an input (e.g. `NumberShots` in `MP2AGE`). Such metadata
+fields MUST be provided in the accompanying json files.
+* The metadata fields listed in the OPTIONAL column can be used to form different flavors
+of an existing file collection suffix, despensing with the need for introducing a new
+suffix. See _Deriving the intended qMRI application from an ambiguos file collection_
+section for details.
+
+#### Field maps
+
+| **File collection**     | **REQUIRED metadata**                                                                                |
+|-------------------------|------------------------------------------------------------------------------------------------------|
+| TB1DAM                  | `FlipAngle`                                                                                          |
+| TB1EPI                  | `EchoTime`, `FlipAngle`, `TotalReadoutTime`, `MixingTime`                                            |
+| TB1AFI                  | `RepetitionTime`                                                                                     |
+| TB1TFL                  |                                                                                                      |
+| TB1RMF                  |                                                                                                      |
+| TB1SRGE<sup>*</sup>     | `FlipAngle`, `InversionTime`, `RepetitionTimeExcitation`, `RepetitionTimePreperation`, `NumberShots` |
+| RB1COR                  |                                                                                                      |
+
+<sup>*</sup> Please see TB1SRGE-specific notes for the calculation of `NumberShots`.
+
+### Meta-data requirements for qMRI maps
+
+As qMRI maps are stored as derivatives, they are subjected to the metadata requirements of 
+[derived datasets](03-modality-agnostic-files.html#derived-dataset-and-pipeline-description).
+
+An example `dataset_description.json` for a qMRI map derivatives folder: 
+
+```
```

Comment:
for code blocks that have no specific syntax such as python, or json, .. please use "text"

```suggestion
```text
```

Reply:
https://github.com/bids-standard/bids-specification/pull/690/commits/274e1f82eade7d6d74d59bb6b5d9ebcf0b4c34a4